sui/its.js

```diff
@@ -106,26 +86,26 @@ if (require.main === module) {
 
     // This command is used to setup the trusted address on the InterchainTokenService contract.
     // The trusted address is used to verify the message from the source chain.
-    const setupTrustedAddressProgram = new Command()
-        .name('setup-trusted-address')
-        .command('setup-trusted-address <trusted-chain>')
+    const setupTrustedChainsProgram = new Command()
+        .name('add-trusted-chains')
+        .command('add-trusted-chains <trusted-chain>')
```

Comment:
```suggestion
        .command('add-trusted-chains <trusted-chains...>')
```
this should parse it as an array, you can test